### ISSUE TITLE
[Bug]: Wireguard setup fails when activating egress on a node 

### ISSUE THREAD

**Issue by Typhonragewind** [gravitl/netmaker#1621]
### What happened?

Still learning the ropes of netmaker, so maybe i'm doing something dumb.
I created a network, "wireguard" and activated both ingress and egress for it, so I can use wireguard on my phone. However, when i add the egress routes "0.0.0.0/0,::/0" the wg-quick command fails due to the pre/post-routing scripts.

### Version

v0.16.0

### What OS are you using?

Linux

### Relevant log output

```shell
[netmaker] 2022-09-30 12:20:48 Warning: `/etc/netclient/config/nm-wireguard.conf' is world accessible

[#] ip link add nm-wireguard type wireguard

[#] wg setconf nm-wireguard /dev/fd/63

[#] ip -4 address add 10.51.112.254 dev nm-wireguard

[#] ip link set mtu 1280 up dev nm-wireguard

[#] `iptables -A FORWARD -i nm-wireguard -j ACCEPT ; iptables -A FORWARD -o nm-wireguard -j ACCEPT ; iptables -t nat -A POSTROUTING -o nm-wireguard -j MASQUERADE ; ; iptables -A FORWARD -i nm-wireguard -j ACCEPT ; iptables -A FORWARD -o nm-wireguard -j ACCEPT ; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE ; `

/usr/bin/wg-quick: command substitution: line 295: syntax error near unexpected token `;'

/usr/bin/wg-quick: command substitution: line 295: `iptables -A FORWARD -i nm-wireguard -j ACCEPT ; iptables -A FORWARD -o nm-wireguard -j ACCEPT ; iptables -t nat -A POSTROUTING -o nm-wireguard -j MASQUERADE ; ; iptables -A FORWARD -i nm-wireguard -j ACCEPT ; iptables -A FORWARD -o nm-wireguard -j ACCEPT ; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE ; '

[#] ip link delete dev nm-wireguard (11)
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by Typhonragewind** [comment]
It seems my issue search wasn't thorough enough. Sorry for bothering and thanks for the quick reply!

**Comment by mattkasun** [comment]
fix will be in 0.16.1
for time being is you first create the egress followed by ingress, the problem will not happen
